<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="image_width" value="160" />
    <param name="image_height" value="120" />
    <param name="video_device" value="/dev/video1" />
    <param name="pixel_format" value="yuyv" /><!--"mjpeg" "yuyv"-->
    <param name="camera_frame_id" value="usb_cam" />
    <param name="framerate" value="20" />
    <!--<param name="io_method" value="mmap"/>-->
    <!--<param name="camera_info_url" type="string" value="file://$(find usb_cam)/usb_cam.yaml" />-->
    
  </node>
  <!--<node name="image_proc" pkg="image_proc" type="image_proc" ns="usb_cam">--><!--/>-->
    <!--<param name="camera_info_url" value="file:///cal.yml" />
  </node>-->
  <!--<node name="image_proc" pkg="image_proc" type="image_proc">
            <remap from="image_raw" to="usb_cam/image_raw" />
            <remap from="camera_info" to="usb_cam/camera_info" /></node>-->
</launch>

<!--
<param name="image_width" value="640" />
<param name="image_height" value="480" />
<launch>
    <group name="cam_ptam">
        <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" />

        <node name="image_proc" pkg="image_proc" type="image_proc">
            <remap from="image_raw" to="usb_cam/image_raw" />
            <remap from="camera_info" to="usb_cam/camera_info" />
        </node>

        <node name="ptam" pkg="ptam" type="ptam" clear_params="true" output="screen">
            <remap from="image" to="image_mono" />
            <remap from="pose" to="pose"/>
            <rosparam file="$(find ptam)/PtamFixParams.yaml"/>
        </node>
    </group>
</launch>
<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
<remap from="image" to="/usb_cam/image_raw"/>
<param name="autosize" value="true" />
</node>
"/usb_cam/image_raw" "/usb_cam/image_mono"
-->
